### DISCUSSION TITLE
Accessing buckets and files in MinIO

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#9137] [Help]
Hi there, I managed to gain access to MinIO in my self hosted BudiBase with the help of your docs ( https://docs.budibase.com/docs/accessing-minio )

Now I can create new buckets, and upload files to these buckets, but I can no see/read any existing buckets.

I'm also wondering whether I could link the files in buckets to records in SQL data sources.

**Comment by melohagan** [comment]
Hey @ecxs 

I don't think this is currently possible - I think you've already come across this issue: https://github.com/Budibase/budibase/issues/4302

**Reply by ecxs** [reply]
Hi @melohagan although slightly different (issue #4302 is about supporting attachments/blobs on external SQL data sources), it's basically the same issue, which is being able to view, upload and download files/documents. For me, this is really essential, as the majority of my web portals needs this.
